enterprise/models.py

```diff
@@ -807,6 +807,30 @@ def __repr__(self):
         return self.__str__()
 
 
+@python_2_unicode_compatible
+class EnterpriseEnrollmentSource(TimeStampedModel):
+    """
+    Define a Name and Source for all Enterprise Enrollment Sources.
+
+    .. no_pii:
+    """
+
+    MANUAL = 'manual'
+    API = 'enterprise_api'
+    ENROLLMENT_URL = 'enrollment_url'
+    CODE_REDEMPTION = 'code_redemption'
```

Comment:
nit: i believe both offers and codes will end up getting mapped to this condition, so this might be better named as offer_redemption, as codes are really just a mechanism to apply offers.